My-Spec-Tutorial.package/MyFirstWindow.class/README.md

```diff
@@ -0,0 +1 @@
+MyFirstWindow open
```

Comment:
It's nice to abstract the (in this case Spec) implementation details and give users a simple message here.